src/components/Card.js

```diff
@@ -47,6 +46,14 @@ export default class Card {
     this._cardElement.querySelector("#el-card-image").src = this._link;
     this._cardElement.querySelector("#el-card-image").alt = this._name;
     this._cardElement.querySelector("#el-card-title").textContent = this._name;
+    //
+    if (this._isLiked) {
+      this._cardElement
+        .querySelector(".element__like-button")
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.

```js
  getView() {
    this._element = this._getTemplate();
    this._likeButton = this._element.querySelector(".card__like-button");
    this._cardImage = this._element.querySelector(".card__image");
    this._trashButton = this._element.querySelector(".card__trash-button");

```